src/app/src/util/embeddedMap.js

```diff
@@ -49,12 +49,13 @@ export const formatExistingFields = (fields = []) =>
 const doesExist = (field, existingFields) =>
     existingFields.some(f => f.columnName === field);
 
-// Keys in this object must be kept in sync with the NonstandardField.DEFAULT_FIELDS in django/api/models.py
+// Keys in this object must be kept in sync with the NonstandardField.EXENDED_FIELDS in django/api/models.py
```

Comment:
Note that although we have implemented support for native language name it is not part of the publicly acknowledged lists of extended fields that will be available when the extended profile feature is launched. As a result, we do not included it in NonstandardField.EXTENDED_FIELDS.